### PR TITLE
Add page for Firefox Differentiators 

### PR DESCRIPTION
**Pull request by caitmuenster**
Migrating [Firefox differentiators](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Firefox_differentiators) to Extension Workshop.

### CODE REVIEW

**Comment by caitmuenster** [comment]
Hi @muffinresearch could you review this when you have a few minutes?

File: documentation/develop/firefox-differentiators.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    aside=""
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Help users separate their browser activities with [Contextual Identities](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/contextualIdentities) a.k.a. Containers. Give groups of tabs their own local browser storage: cookie store and local data in localStorage and indexedDB. This keeps details such as site preferences, signed in sessions, and ad tracking data private to the tabs.
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:45]:
```suggestion
Help users separate their browser activities with [Contextual Identities](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/contextualIdentities) a.k.a. Containers. Give groups of tabs their own local browser storage: cookie store and local data in localStorage and indexedDB. This keeps details such as site preferences, signed in sessions, and ad tracking data private to the tabs.
```

File: documentation/develop/firefox-differentiators.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Record and display additional or complementary information about a web page with the [Sidebar](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Sidebars) UI component, a panel that sits alongside web pages.
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:67]:
```suggestion
Record and display additional or complementary information about a web page with the [Sidebar](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/user_interface/Sidebars) UI component, a panel that sits alongside web pages.
```

File: documentation/develop/firefox-differentiators.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Help users locate content of interest in a web page, with the ability to [find and highlight text in a webpage](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/find).
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:111]:
```suggestion
Help users locate content of interest in a web page, with the ability to [find and highlight text in a webpage](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/find).
```

File: documentation/develop/firefox-differentiators.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Perform searches directly from your extension, with features to [get details of search engines and perform a search](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/search).
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:133]: Same here please

File: documentation/develop/firefox-differentiators.md [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Add an extra dimension to your themes by [updating the browser theme at runtime](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/theme). With this feature you can alternate themes by time of day, apply a different theme when users browse your website, reflect user actions in your extension, and more.
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:155]: Same here please

File: documentation/develop/firefox-differentiators.md [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Provide users with control over their browsing experience from your extension with the ability to [modify (some) global browser settings](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserSettings).
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:177]: Same here please

File: documentation/develop/firefox-differentiators.md [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Enable users to reliably add your extension’s features to new pages and sites, with the ability to [register and unregister content scripts at runtime](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/contentScripts).
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:199]: Here too

File: documentation/develop/firefox-differentiators.md [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Take advantage of external services to handle protocols or content types by registering [protocol handlers](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/protocol_handlers) in the `manifest.json`.
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:221]: Here too :)

File: documentation/develop/firefox-differentiators.md [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+	aside=""
+%}
+
+<!-- END: Two Column Body Module -->
+
+<!-- Two Column Body Module -->
+
+{% capture col_1_content %}
+
+Perform additional web tasks such as [resolving domain names](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/dns) and [proxying web requests](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy).
```
Comment (championshuttler) [inline documentation/develop/firefox-differentiators.md:243]: Here as well

**Comment by caitmuenster** [comment]
Thanks @championshuttler & @muffinresearch! 🙏🏼 This page is now called "Unique Firefox Capabilities" (though I will keep thinking about other names that might work better -- I am open to suggestions!).

**Pull request merged**